homeassistant/components/ecoforest/config_flow.py

```diff
@@ -0,0 +1,108 @@
+"""Config flow for Ecoforest integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from httpx import BasicAuth
+from pyecoforest.api import EcoforestApi
+from pyecoforest.exceptions import EcoforestAuthenticationRequired
+from pyecoforest.models.device import Device
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN, MANUFACTURER
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST, default="https://192.168.1.1:8000"): str,
```

Comment:
Default can be omitted, everyone's network is built differently, and I would imagine that someone working with HA at least know what a host is.

Reply:
The reason why I considered adding the default was to provide a hint for the https port, but I will add this to the documentation, it seems a better place.